Crash in MainActivity

It looks like there's a bug in this code in MainActivity:
```
    public void updateBTarmac()
    {
        if(isAuto) {
            bTarmac.setBackgroundColor( (zAutoTarmac == 0) ? darkThemeRed : darkThemeGreen);
            bTarmac.setText(climbDisplayText[zAutoTarmac]);
        }
        else {
            bTarmac.setBackgroundColor( (zTarmac == 0) ? darkThemeRed : darkThemeGreen);
            bTarmac.setText(climbDisplayText[zTarmac]);
        }
    }
```
The problem seems to be that `zAutoTarmac` and `zTarmac` are not always valid indexes for the `climbDisplayText` array. The size of this array is 3, but the index is too large at some point (3).

crash:
```

2024-02-01 20:27:39.967 11498-11498 AndroidRuntime          com.example.a2024frcscoutingapp      E  FATAL EXCEPTION: main
                                                                                                    Process: com.example.a2024frcscoutingapp, PID: 11498
                                                                                                    java.lang.IllegalStateException: Could not execute method for android:onClick
                                                                                                    	at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:446)
                                                                                                    	at android.view.View.performClick(View.java:5047)
                                                                                                    	at android.view.View$PerformClick.run(View.java:20286)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:739)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                                                    	at android.os.Looper.loop(Looper.java:135)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:5597)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at java.lang.reflect.Method.invoke(Method.java:372)
                                                                                                    	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:984)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
                                                                                                    Caused by: java.lang.reflect.InvocationTargetException
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at java.lang.reflect.Method.invoke(Method.java:372)
                                                                                                    	at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:441)
                                                                                                    	at android.view.View.performClick(View.java:5047) 
                                                                                                    	at android.view.View$PerformClick.run(View.java:20286) 
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:739) 
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:95) 
                                                                                                    	at android.os.Looper.loop(Looper.java:135) 
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:5597) 
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                                    	at java.lang.reflect.Method.invoke(Method.java:372) 
                                                                                                    	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:984) 
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779) 
                                                                                                    Caused by: java.lang.ArrayIndexOutOfBoundsException: length=3; index=3
                                                                                                    	at com.example.a2024frcscoutingapp.MainActivity.updateBTarmac(MainActivity.java:450)
                                                                                                    	at com.example.a2024frcscoutingapp.MainActivity.clickTarmac(MainActivity.java:438)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                                    	at java.lang.reflect.Method.invoke(Method.java:372) 
                                                                                                    	at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:441) 
                                                                                                    	at android.view.View.performClick(View.java:5047) 
                                                                                                    	at android.view.View$PerformClick.run(View.java:20286) 
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:739) 
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:95) 
                                                                                                    	at android.os.Looper.loop(Looper.java:135) 
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:5597) 
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                                    	at java.lang.reflect.Method.invoke(Method.java:372) 
                                                                                                    	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:984) 
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779) 
```

--- a/app/src/main/java/com/example/a2024frcscoutingapp/MainActivity.java
+++ b/app/src/main/java/com/example/a2024frcscoutingapp/MainActivity.java
@@ -434,11 +434,11 @@ public class MainActivity extends AppCompatActivity {
 
     public void clickTarmac(View v){
         if(isAuto){
-            zAutoTarmac = (zAutoTarmac + 1)%4;
+            zAutoTarmac = (zAutoTarmac + 1)%3;
             updateBTarmac();
         }
         else{
-            zTarmac = (zTarmac + 1)%4;
+            zTarmac = (zTarmac + 1)%3;
             updateBTarmac();
         }
     }

